test/application_system_test_case.rb

```diff
@@ -65,14 +71,10 @@ def fill_autocomplete(name, options = {})
   end
 
   def any_checked?(selector)
-    page.find_all(selector).reduce do |a, b|
-      a.checked? || b.checked?
-    end
+    page.find_all(selector).any?(&:checked?)
```

Comment:
I think I didn't fully understand here the difference between `(selector)` and `checked` and `picked`

Reply:
cleared